src/server.cpp

```diff
@@ -239,6 +246,11 @@ Status Server::startModules(ovms::Config& config) {
     // GRPC & REST
     Status status;
     bool inserted = false;
+#if (PYTHON_DISABLE == 0)
+    auto itPython = modules.end();
```

Comment:
move:
auto it = modules.end() above ifdef and dont introduce another iterator since we don't use it later anyway.